Modules/time.js

```diff
@@ -0,0 +1,5 @@
+import { DateTime } from './luxon.js';
+
+const showDateAndTime = document.querySelector('#date');
+
+showDateAndTime.innerHTML = DateTime.now().toLocaleString(DateTime.DAYTIME_MED);
```

Comment:
You did great by using `'uxon.js' ` to display the date as required in the project but make sure the date is visible in the user interface as you see in the picture below  of your design the date is visible kindly respect the instruction below

The current date is displayed below the navigation bar, for this requirement, students **should** use the 
[Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time)

* Package is added as a .js file to the project (in full or minified version)
* Module is imported `import { DateTime } from "luxon";

## Your design 

![image](https://user-images.githubusercontent.com/95297251/206199898-3dc6e034-c4ae-4fb1-9c50-b48a365e2da5.png)


## Design with the date 

![image](https://user-images.githubusercontent.com/95297251/206200209-969a6999-b0cc-4b67-9e51-bc06521f1075.png)